crates/assistant/src/inline_assistant.rs

```diff
@@ -1766,6 +1766,7 @@ impl PromptEditor {
                             move |settings, _| settings.set_model(model.clone()),
                         );
                     },
+                    ModelType::Default,
```

Comment:
Inline assistant 1 still uses default model because of how it extends the conversation